File: app-service-comments/src/main/java/com/github/arhor/aws/graphql/federation/comments/data/repository/UserRepresentationRepository.java
package com.github.arhor.aws.graphql.federation.comments.data.repository;

import com.github.arhor.aws.graphql.federation.comments.data.model.UserRepresentation;
import org.springframework.data.repository.CrudRepository;

import java.util.UUID;

public interface UserRepresentationRepository extends CrudRepository<UserRepresentation, UUID> {
}

File: app-service-votes/src/main/kotlin/com/github/arhor/aws/graphql/federation/votes/data/repository/UserRepresentationRepository.kt
package com.github.arhor.aws.graphql.federation.votes.data.repository

import com.github.arhor.aws.graphql.federation.votes.data.model.UserRepresentation
import org.springframework.data.repository.CrudRepository

import java.util.UUID

interface UserRepresentationRepository : CrudRepository<UserRepresentation, UUID>
